test/azure_devops_agent_aci_test.go

```diff
@@ -284,12 +291,90 @@ func TestDeployAzureDevOpsLinuxAgentsIntoExistingRresourceGroup(t *testing.T) {
 			t.Fatalf("Test failed. Expected number of agents is %d. Actual number of agents is %d", expectedAgentsCount, actualAgentsCount)
 		}
 	})
+}
+
+// This function tests the deployment of Azure DevOps Linux and Windows agents
```

Comment:
I see only Linux agents in the fixture. No?

Reply:
Copy paste issue.

Reply:
So there is not test for Windows agent pool cleanup? Is this scenario supported by the PR? If so, it would be great to have a test to validate it also cleans up Windows pool.

Reply:
They cleanup only works for linux

Reply:
Why there is some update on the windows script then? What is blocking for having it working on Windows?